gdx/src/com/badlogic/gdx/graphics/g2d/SpriteCache.java

```diff
@@ -241,7 +243,7 @@ public int endCache () {
 	/** Invalidates all cache IDs and resets the SpriteCache so new caches can be added. */
 	public void clear () {
 		caches.clear();
-		((Buffer) mesh.getVerticesBuffer()).clear().flip();
+		((Buffer) mesh.getVerticesBuffer(true)).clear().flip();

```

Comment:
`.clear()` and `.flip()` should set the buffer as dirty , not `getBuffer()`

Reply:
we can't modify clear/flip methods, those are from JDK.